arch/platform/zoul/firefly-reva/board.h

```diff
@@ -99,9 +99,18 @@
  */
 /*---------------------------------------------------------------------------*/
 /* In leds.h the LEDS_BLUE is defined by LED_YELLOW definition */
-#define LEDS_GREEN    (1 << 4) /**< LED1 (Green) -> PD4 */
-#define LEDS_BLUE     (1 << 3) /**< LED2 (Blue)  -> PD3 */
-#define LEDS_RED      (1 << 5) /**< LED3 (Red)   -> PD5 */
+#define LEDS_GREEN_PIN      4
+#define LEDS_GREEN    (1 << LEDS_GREEN_PIN) /**< LED1 (Green) -> PD4 */
+
+#define LEDS_BLUE_PIN       3
+#define LEDS_BLUE     (1 << LEDS_BLUE_PIN)  /**< LED2 (Blue)  -> PD3 */
+
+#define LEDS_RED_PIN        5
+#define LEDS_RED      (1 << LEDS_RED_PIN)   /**< LED3 (Red)   -> PD5 */
```

Comment:
should this use `gpio_hal_pin_to_mask`?

Reply:
I'd rather not at this stage:
* Semantically those `LEDS_RED_PIN` are not of type `gpio_hal_pin_t` but a magic number used internally by the port's LED driver.
* Plus, this is fundamentally a configuration file that is meant to help configure `gpio-hal.h` and I don't want to mess about with circular include situations now that this is basically ready.
* Plus I've got something else in mind for LEDs... :)

Reply:
ok sure!